src/components/InstancesList/InstancesList.js

```diff
@@ -1174,11 +1173,15 @@ class InstancesList extends React.Component {
     const shortcuts = [
       {
         name: 'new',
-        handler: handleKeyCommand(() => {
+        handler: (e) => {
+          const nodeName = e.target?.nodeName || '';
+          const editTextField = (nodeName === 'TEXTAREA') || (nodeName === 'INPUT');
```

Comment:
Let's rename it to `isEditTextField` to be consistent with boolean flag naming

Reply:
Yeah sure